patches/015-fix-linkage.patch

```diff
@@ -0,0 +1,25 @@
+From 8b36034dd8a69ffb7ec7e092b78171bd635ce345 Mon Sep 17 00:00:00 2001
```

Comment:
Actually, can you remove this patch header and just include the diff?

Reply:
Done